extension/package.json

```diff
@@ -1,25 +1,21 @@
 {
   "name": "@shopgate/ext-magento-favorites",
-  "version": "0.1.0",
-  "description": "prototype",
+  "version": "1.0.0",
+  "description": "Helps with retrieving wishlist from Magento. Relies on @shopgate/magento-cart & @shopgate/favorites extensions to operate",
   "scripts": {
     "lint": "./node_modules/.bin/eslint --ignore-path ../.gitignore --ext .js --ext .jsx .",
     "test": "./node_modules/.bin/_mocha --use_strict --recursive ./test/unit",
-    "cover": "./node_modules/.bin/nyc npm test && ./node_modules/.bin/nyc report --reporter=text-lcov | coveralls"
+    "cover": "./node_modules/.bin/nyc npm test && ./node_modules/.bin/nyc report --reporter=text-lcov | coveralls",
+    "newman": "newman run ./test/newman/collection.json --globals ./test/newman/globals.json --environment ./test/newman/environment.json"
```

Comment:
Please add newman to the devDependencies + call it from the node_modules folder.

Reply:
oops, forgot that I have it globally